dev/AppNotifications/ShellLocalization.cpp

```diff
@@ -246,7 +247,9 @@ HRESULT Microsoft::Windows::AppNotifications::ShellLocalization::DeleteIconFromC
     std::wstring notificationAppId{ Microsoft::Windows::AppNotifications::Helpers::RetrieveNotificationAppId() };
 
     // path: C:\Users\<currentUser>\AppData\Local\Microsoft\WindowsAppSDK\{AppGUID}.png
-    std::path iconFilePath{ RetrieveLocalFolderPath() / (notificationAppId + c_pngExtension) };
+    std::path iconFilePath{ RetrieveLocalFolderPath() };
+    iconFilePath /= notificationAppId;
```

Comment:
Can we do this iconFilePath = (iconFilePath / notificationAppId) + c_pngExtension?

Reply:
No, because there is no defined operator+ for std::filesystem::path. The original operator combination works because it does `path / (string + string)` = `path / string`

Reply:
Ok. SO if the original worked in a single line, why are we changing this?

Reply:
I reverted it to a single line.